docs/reference/Admin_Actions.md

```diff
@@ -4,6 +4,30 @@ When you consider that most backend services are a black box of code and decisio
 
 With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.
 
+## 2023 - April 15
+
+### Uber-Juno
+
+The [Uber-Juno](https://github.com/JunoLab/uber-juno) package uses an older unsupported coding style, sometimes called JavaScript Sloppy Mode, that's no longer supported in newer versions of NodeJS. Meaning the package crashes as soon as installed, and cannot be used without the user manually editing the source code.
```

Comment:
as per the observation by comment https://github.com/pulsar-edit/pulsar/issues/482#issuecomment-1506542131 , it said in that error the following ...

> Modules are automatically in strict mode.

and since no issues are shown after that change, I think "it uses [Sloppy Mode](https://developer.mozilla.org/en-US/docs/Glossary/Sloppy_mode)" no longer applies...